src/utils/miscellaneous.ts

```diff
@@ -13,7 +13,28 @@ export function assertNonNull<T>(val: T, message: string): asserts val is NonNul
 function noop(..._args: any[]): void {}
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const log = process.env.NODE_ENV === 'test' ? noop : (...args: any[]): void => console.log(...args)
+export const logInfo = process.env.NODE_ENV === 'test' ? noop : (...args: any[]): void => console.log(...args)
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let debugEnabled = process.env.NODE_ENV === 'development'
+
+declare global {
+  interface Window {
+    toggleDebug: () => void
+  }
+}
+
+window.toggleDebug = (): boolean => {
```

Comment:
Do we wan't this available in production too?

Reply:
Yes, that's the main reason I added that. It allows us to see the message we see today in a minimized version, only difference, logs are disabled by default

Reply:
Cool then